app/src/main/res/values/colors.xml

```diff
@@ -18,6 +18,7 @@
     <color name="onboarding_button_higlight_stroke">@color/photonBlue50</color>
 
     <color name="browser_overlay_background">@color/photonGrey70</color>
+    <color name="hint_bar_background">#0C0C0D</color>
```

Comment:
This is color/photonGrey90, included in AC probably - in general, all colors we use are from the color palette.